modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts

```diff
@@ -110,49 +119,24 @@ export function StyleSheetLoader(documentOrShadowRoot: Document | ShadowRoot, se
 
     // Workaround for WebKit that doesn't properly support the onload event for link elements
     // Or WebKit that fires the onload event before the StyleSheet is added to the document
-    const waitForWebKitLinkLoaded = function () {
-      wait(function () {
+    const waitForWebKitLinkLoaded = () => {
+      wait(() => {
         const styleSheets = documentOrShadowRoot.styleSheets;
-        let styleSheet, i = styleSheets.length, owner;
+        let i = styleSheets.length;
 
         while (i--) {
-          styleSheet = styleSheets[i];
-          owner = styleSheet.ownerNode ? styleSheet.ownerNode : styleSheet.owningElement;
-          if (owner && owner.id === link.id) {
+          const styleSheet = styleSheets[i];
+          const owner = styleSheet.ownerNode;
+          if (owner && (owner as Element).id === link.id) {
```

Comment:
This possibly should have a guard, however `id` will just be undefined if it's not an element so adding the guard would just end up in bloat which is why I opted for the cast here.